scripts/templates/admin.html

```diff
@@ -0,0 +1,41 @@
+<head>
+  <title>admin page</title>
+</head>
+
+<body>
+  <div id="userlist">
+    <table>
+      <tr>
+        <th>user</th>
+        <th>(❌ = b&amp;)</th> 
+      </tr>
+        {% for username, ban_status in all_users %}
+            <tr>
+               <th> {{ username }} </th>
+                {% if ban_status %}
+                   <td> <a href="/admin/unban/{{ username }}">❌</a></td>
+                {% else %}
+                   <td> <a href="/admin/ban/{{ username }}">🔨</a></td>
```

Comment:
I think these icons are the wrong way around.  ❌ looks negative, so I'd read it as "disable this account".  Just using words is probably clearest here.  "disable" and "enable" would do fine.